.env.example

```diff
@@ -12,11 +12,14 @@ KEYCLOAK_URL=http://localhost:8080/auth
 KEYCLOAK_REDIRECT_URI=http://localhost:4000/auth
 STRIPE_PUBLIC_API_KEY=ENTER_STRIPE_PUBLIC_KEY_HERE
 OAUTH2_AUTH_URL=http://localhost:4444/oauth2/auth
-OAUTH2_TOKEN_URL=http://hydra:4444/oauth2/token
+OAUTH2_TOKEN_URL=http://hydra.auth.reaction.localhost:4444/oauth2/token
 OAUTH2_CLIENT_ID=reaction-next-starterkit
 OAUTH2_CLIENT_SECRET=CHANGEME
 OAUTH2_REDIRECT_URL=http://localhost:4000/callback
 OAUTH2_IDP_HOST_URL=http://reaction.api.reaction.localhost:3000/
+OAUTH2_HOST=hydra.auth.reaction.localhost
+OAUTH2_ADMIN_PORT=4445
+PASSPORT_SESSION_SECRET=CHANGEME
```

Comment:
hi @focusaurus, I already approved the PR (it works after your last push). Then I noticed that this PASSPORT_SESSION_SECRET env is not being used anywhere. Is there a reason it's added here? 
Just wanted to confirm that before merging

Reply:
Looks like that was renamed in a separate piece of work Eric D was doing in 7eef1813f08fecc98b03660e450840af959c8e30. I'll remove it. It is now just called SESSION_SECRET which we have.